### ISSUE TITLE
Issues with Ragtime and Heroku

### ISSUE THREAD

**Issue by kouphax** [weavejester/ragtime#46]
I've recently started a new project (Luminus based, nothing added) and added ragtime `0.3.7` to perform migrations.  This worked (past tense see later) locally as expected however when I deploy to Heroku and run `heroku run "lein ragtime migrate"` it fails with the following error,

```
Exception in thread "main" java.lang.RuntimeException: No such var: sql/with-connection, compiling:(ragtime/sql/database.clj:22:3)
```

I suspect this is related to #23 but I haven't been able to find a solution.  In a bid to diagnose the issue locally I blew away my `~/.m2` directory and ran the command locally and now I'm getting the same issue.

**UPDATE** It seems I can workaround the issue both locally and on heroku by issuing

```
lein do clean, ragtime migrate
```

However I'm not sure if this is just covering up a dependency problem so thought it worth mentioning.

**Comment by upgradingdave** [comment]
fwiw, I just ran into this same stack trace using ragtime on a local luminus project (I'm not using anything related to heroku). 

I can reproduce consistently by first doing: 

`lein uberjar`

and then 

`lein ragtime migrate`

If I do a `lein do clean, ragtime migrate`, then I don't see the error. 

The project was generated using the luminus lein template with http-kit and postgres profiles. And I also added pallet to manage deployments.

**Comment by mll** [comment]
Hi! I have exactly the same issue on heroku, but unfortunately 'lein do clean, ragtime migrate' doesn't help in my case. Help!!!!

**Comment by mll** [comment]
Ok, so I seem to know what is going on. It seems that there's a problem on maven repo with [org.clojure/java.jdbc "0.3.6"]

Add [org.clojure/java.jdbc "0.3.7"] to your project.clj's :requirements section. This solved the issue for me.

**Comment by weavejester** [comment]
Should be fixed in Ragtime 0.4.0 and above.
